src/99-appendices/06-meg-file-formats.md

```diff
@@ -156,11 +156,11 @@ More about the 4D neuroimaging/BTi data organization at:
 ## KIT/Yokogawa/Ricoh
 
 Each experimental run on a KIT/Yokogawa/Ricoh system yields a raw (`.sqd`,
-`.con`) file with its associated marker coil file (`.mrk`), which contains coil
-positions in the acquisition system’s native space. Head points and marker
-points in head space are acquired using third-party hardware. One SHOULD
-rename/create a father run specific directory and keep the original files for
-each run inside.
+`.con`) file with its associated marker coil file(s) (`.sqd`, `.mrk`), which
+contains coil positions in the acquisition system’s native space. Head points
+and marker points in head space are acquired using third-party hardware. One
+SHOULD rename/create a further run specific directory and keep the original
+files for each run inside.
```

Comment:
this is colliding with #19 to some extend. Could you review that PR @monkeyman192?

Reply:
I'll wait for that PR to go through then rebase this onto the master to pick up the changes with the suggested rewording by @robertoostenveld